src/main/scala/hmda/parser/ParserErrorModel.scala

```diff
@@ -0,0 +1,22 @@
+package hmda.parser
+
+import com.typesafe.config.ConfigFactory
+
+object ParserErrorModel {
+
+  val config = ConfigFactory.load()
+  val expectedLegth = config.getInt("hmda.filing.ts.length")
+
+  def notAnInteger(fieldName: String) = s"$fieldName is not an integer"
+
+  trait ParserValidationError {
+    def errorMessage: String
+  }
+
+  case class IncorrectNumberOfFields(length: Int)
```

Comment:
I'm just curious, why is this error modeled as a case class and the others as case object?  

Is this object specific to the TS length?  It seems like it can be reused for the LAR, with some changes to the config

Reply:
This needs to be a case class because it takes a constructor parameter. Case objects (or traits) can't take parameters, compilation will fail.